LICENSE

```diff
@@ -1,6 +1,6 @@
 MIT License
 
-Copyright (c) 2018 XamarinCommunityToolkit
+Copyright (c) 2018 Xamarin.Toolkit
```

Comment:
Should this be Microsoft? Or https://github.com/Microsoft/UWPCommunityToolkit/blob/master/license.md

Reply:
Good question...

Reply:
Microsoft?, Xamarin Inc?, Xamarin.Toolkit?
In UWP Community Toolkit is "NET Foundation and Contributors".

Reply:
yeah, i think that is what we want. I need to chat with the .NET Foundation team @jongalloway